### PR TITLE
[Bug] SingleLineAfterImportsFixer - no meaningful after open semicolon

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by SpacePossum** [comment]
replaces https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1961

**Comment by SpacePossum** [comment]
green so ready?

File: Symfony/CS/Tests/IntegrationTest.php [hunk 1]
```diff
@@ -35,4 +37,99 @@ protected static function getTempFile()
...
+        );
+    }
+
+    /**
+     * @param string   $source
+     * @param string[] $whiteListFixerNames
+     *
+     * @dataProvider provideBasicSources
+     */
+    private function doTest($source, array $whiteListFixerNames = array())
```
Comment (keradus) [inline Symfony/CS/Tests/IntegrationTest.php]: Please, could you provide some information for that method?
on 2.x we have it: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Test/AbstractIntegrationTestCase.php#L152

but I don't think they are combatible, am I right ?

**Comment by SpacePossum** [comment]
Updated, I've moved the test to its own file.
Please note that the test relies on some fixer names, unfortunately the following have been renamed on master (so please be aware when merging upstream)
```
eof_ending -> single_blank_line_at_eof
short_echo_tag -> no_short_echo_tag
php_closing_tag -> no_closing_tag
```

File: Symfony/CS/Tests/BasicFileFixingTest.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests;
+
+use Symfony\CS\Fixer;
+
+/**
+ * Test the behavior of all fixers on some basic PHP files.
```
Comment (keradus) [inline Symfony/CS/Tests/BasicFileFixingTest.php]: I can't get the purpose of this test

Comment (SpacePossum) [inline Symfony/CS/Tests/BasicFileFixingTest.php]: Maybe it no longer has a use, but given the bugs found and fixed in this PR and this one; https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1978 the test had a purpose.
I'll delete it

Comment (keradus) [inline Symfony/CS/Tests/BasicFileFixingTest.php]: I just does too much at same time. And previously it was IntegrationTest right ?

**Comment by keradus** [comment]
@SpacePossum why closing this one ?

**Comment by SpacePossum** [comment]
Sorry forgot the ref.: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1989
I was making conflicts in my own PR's...

**Comment by keradus** [comment]
Great, thanks !

**Pull request closed without merging**